src/framework/mpas_dmpar.F

```diff
@@ -8463,7 +8463,11 @@ subroutine mpas_dmpar_exch_group_pack_buffer_field1d_integer(exchangeGroup, fiel
       commListPtr => exchangeGroup % sendList
       if (.not. associated(commListPtr)) return
       commListSize = commListPtr % commListSize
+#ifdef CPRPGI
```

Comment:
Would be good to add a single comment line here, what CPRPGI is and why it is needed.

Reply:
Updated with a comment below.

Reply:
Thanks!